### PR TITLE
[TASK] Switch to confvals for mod.web_layout

### PR DESCRIPTION
**Pull request by linawolf**
* use confvals
* correct indentation
* correct header levels
* Give headers to examples

No other text changes made.

The page did not yet exist in 11.5

releases: main, 12.4

### CODE REVIEW

File: Documentation/PageTsconfig/Mod/WebLayout.rst [hunk 1]
```diff
@@ -1,327 +1,350 @@
...
+                                    colPos = 3
+                                    colspan = 1
+                                }
+                            }
                         }
+                        2 {
+                            columns {
+                                1 {
+                                    name = Main
+                                                colPos = 0
```
Comment (brotkrueml) [inline Documentation/PageTsconfig/Mod/WebLayout.rst:93]: indentation is here wrong. Also closing brackets later.

File: Documentation/PageTsconfig/Mod/WebLayout.rst [hunk 2]
```diff
@@ -1,327 +1,350 @@
...
+    Content elements; Preview definition
+..  _pageweblayoutpreview:
 
-:aspect:`Default`
-   0
+tt_content.preview
+==================
 
+..  confval:: tt_content.preview
+    :name: mod-web-layout-preview
```
Comment (brotkrueml) [inline Documentation/PageTsconfig/Mod/WebLayout.rst:310]:
```suggestion
    :name: mod-web-layout-tt-content-preview
```

**Review by brotkrueml — COMMENTED** [review]
The page exists, but in "mod" directly and not as a separate file. Was changed recently.

**Comment by linawolf** [comment]
The changes would then have to be backported manually if we really want to put in the work

File: Documentation/PageTsconfig/Mod/WebLayout.rst (hunk 1 above)
Comment (linawolf) [reply Documentation/PageTsconfig/Mod/WebLayout.rst]: Indentation of large files inline rst is hard to handle. Moved the code example to an include file instead.

File: Documentation/PageTsconfig/Mod/WebLayout.rst [hunk 3]
```diff
@@ -1,327 +1,318 @@
...
 
 defLangBinding
 ==============
 
-:aspect:`Datatype`
-   boolean
+..  confval:: defLangBinding
+    :name: mod-web-layout-defLangBinding
+    :type: boolean
+    :Default: false
```
Comment (brotkrueml) [inline Documentation/PageTsconfig/Mod/WebLayout.rst:108]:
```suggestion
    :Default: 0
```

File: Documentation/PageTsconfig/Mod/WebLayout.rst [hunk 4]
```diff
@@ -1,327 +1,318 @@
...
 
 localization.enableCopy
 =======================
 
-:aspect:`Datatype`
-   boolean
+..  confval:: localization.enableCopy
+    :name: mod-web-layout-localization-enableCopy
+    :type: boolean
+    :Default: true
```
Comment (brotkrueml) [inline Documentation/PageTsconfig/Mod/WebLayout.rst:163]:
```suggestion
    :Default: 1
```

File: Documentation/PageTsconfig/Mod/WebLayout.rst [hunk 5]
```diff
@@ -1,327 +1,318 @@
...
 localization.enableTranslate
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+============================
 
-:aspect:`Datatype`
-   boolean
+..  confval:: localization.enableTranslate
+    :name: mod-web-layout-localization-enableTranslate
+    :type: boolean
+    :Default: true
```
Comment (brotkrueml) [inline Documentation/PageTsconfig/Mod/WebLayout.rst:193]:
```suggestion
    :Default: 1
```

File: Documentation/PageTsconfig/Mod/WebLayout.rst [hunk 6]
```diff
@@ -1,327 +1,318 @@
...
+    Buttons; Create new record
 
 noCreateRecordsLink
-~~~~~~~~~~~~~~~~~~~
+===================
+
+..  confval:: noCreateRecordsLink
+    :name: mod-web-layout-noCreateRecordsLink
+    :type: boolean
+    :Default: false
```
Comment (brotkrueml) [inline Documentation/PageTsconfig/Mod/WebLayout.rst:265]:
```suggestion
    :Default: 0
```

**Review by brotkrueml — CHANGES_REQUESTED** [review]
See also about booleans in tsref:
https://docs.typo3.org/m/typo3/reference-typoscript/12.4/en-us/DataTypes/Index.html#boolean

**Review by brotkrueml — APPROVED** [review]

**Pull request merged**